dissect/target/plugins/os/windows/services.py

```diff
@@ -20,6 +20,7 @@
         ("datetime", "ts"),
         ("string", "name"),
         ("wstring", "displayname"),
+        ("wstring", "description"),
```

Comment:
```suggestion
        ("string", "displayname"),
        ("string", "description"),
```
`wstring` is actually just an alias for `string` these days.